packages/backend/src/query/iterateQuery.test.ts

```diff
@@ -194,13 +194,15 @@ describe('iterateQuery', () => {
                 where: {
                     film_id: 1,
                 },
+                hash: '',
```

Comment:
The hash is optional, so this should not be needed. Below in the ArbitratyQueryBuilder.